### PR TITLE
[Docs] Fix mmseg document

### PR DESCRIPTION
**Pull request by AI-Tianlong**
Fix mmseg document
**Before：**
image

**After：**
image

### CODE REVIEW

**Comment by RunningLeon** [comment]
@AI-Tianlong hi, thanks for your PR. 
1. Could sign the [CLA ](https://cla-assistant.io/open-mmlab/mmdeploy?pullRequest=2363)
2. Could fix lint error by running `pip install pre-commit` and `pre-commit run --all-files` and commit new changes.

**Comment by AI-Tianlong** [comment]
> @AI-Tianlong hi, thanks for your PR.
> 
> 1. Could sign the [CLA ](https://cla-assistant.io/open-mmlab/mmdeploy?pullRequest=2363)
> 2. Could fix lint error by running `pip install pre-commit` and `pre-commit run --all-files` and commit new changes.

I'll do it right away

File: docs/zh_cn/04-supported-codebases/mmseg.md [hunk 1]
```diff
@@ -47,7 +47,8 @@ export LD_LIBRARY_PATH=$(pwd)/../mmdeploy-dep/onnxruntime-linux-x64-1.8.1/lib/:$
 
 - 把 `$(pwd)/build/lib` 添加到 `PYTHONPATH`，目的是为了加载 mmdeploy SDK python 包 `mmdeploy_runtime`，在章节 [SDK模型推理](#sdk模型推理)中讲述其用法。
 - 在[使用 ONNX Runtime推理后端模型](#后端模型推理)时，需要加载自定义算子库，需要把 ONNX Runtime 库的路径加入环境变量 `LD_LIBRARY_PATH`中。
-  **方式三：** 源码安装
+
```
Comment (RunningLeon) [inline docs/zh_cn/04-supported-codebases/mmseg.md:50]: pls. remove files in `wokr_dir` git this commit

Comment (AI-Tianlong) [reply docs/zh_cn/04-supported-codebases/mmseg.md]: Done~

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**